src/components/DeliverableView/DocumentCellRenderer.tsx

```diff
@@ -3,20 +3,25 @@ import React from 'react';
 import Link from 'src/components/common/Link';
 import CellRenderer, { TableRowType } from 'src/components/common/table/TableCellRenderer';
 import { RendererProps } from 'src/components/common/table/types';
+import { ENDPOINT_DELIVERABLE_DOCUMENT } from 'src/services/DeliverablesService';
 
 export default function DocumentCellRenderer(props: RendererProps<TableRowType>): JSX.Element {
-  const { column, row, index, value } = props;
+  const { column, row, index } = props;
 
   if (column.key === 'link') {
-    // TODO convert BE label value to locale specific string
     return (
       <CellRenderer
         index={index}
         column={column}
-        // TODO this will be a "BE enum to string converter" once we get the types and enums
         value={
-          <Link to={{ pathname: value as string }} target='_blank'>
-            {row.documentType === 'google' ? 'Google Drive' : ''}
+          <Link
+            to={ENDPOINT_DELIVERABLE_DOCUMENT.replace('{deliverableId}', row.deliverableId).replace(
+              '{documentId}',
+              row.id
+            )}
+            target='_blank'
+          >
+            {row.documentStore}
```

Comment:
I am assuming we don't want to localize these values since they are brand names.

Reply:
We do the same thing for `Terraware`.
If we keep them in strings, allows us to support languages in other scripts. Super theoretical, but totally fine for now. Let's keep it as-is,  we can do a clean sweep later on if there's a need.